### DISCUSSION TITLE
Getting "Unsupported image file type [.]" Error on publishing

### DISCUSSION THREAD

**Discussion by Dowithatie** [Second-Hand-Friends/kleinanzeigen-bot#197] [Q&A] [answered]
When trying to publish an ad, I get the "Unsupported image file type [.]" error:
![image](https://github.com/Second-Hand-Friends/kleinanzeigen-bot/assets/141066786/0f16ac24-dbc6-4480-9336-166245b6c32a)
I think it's because of the way I'm declaring the image in the config file:
![image](https://github.com/Second-Hand-Friends/kleinanzeigen-bot/assets/141066786/4ced77fe-efab-44db-9ef3-b1c1d6e5d426)

pls help :c

**Comment by sebthom** [comment, marked as answer]
1. If you are using backslashes in yaml you probably have to escape them like `C:\\Users\\`. You can also just use single forward slashes. `C:/Users/...`
2. `images: "...Hubschrauber(42020)\.jpg"` looks wrong. What is `\.jpg` supposed to do? Did you mean `images: "...Hubschrauber(42020)\*.jpg"` ?

**Reply by Dowithatie** [reply]
yes, thank you!
